src/database/models/Order.model.ts

```diff
@@ -0,0 +1,44 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from ".";
+
+const Order = sequelize.define(
+  "order",
+  {
+    id: {
+      allowNull: false,
+      autoIncrement: false,
+      primaryKey: true,
+      type: DataTypes.UUID,
+      unique: true,
+      defaultValue: DataTypes.UUIDV4,
+    },
+    userId: {
+      type: DataTypes.UUID,
+      references: {
+        model: "User",
+        key: "id",
+      },
+    },
+    items: {
+      type: DataTypes.JSONB,
+      allowNull: false,
+    },
+    status: {
+      type: DataTypes.ENUM("pending", "shipped", "delivered"),
+      defaultValue: "pending",
```

Comment:
It's not a breaking change, but can you try capitalizing the enums for consistency elsewhere.

e.g `Pending` , `Shipped`, `Delivered`